### PR TITLE
fix nested select (parentSelectorId) when used inside a nested form

### PR DESCRIPTION
**Pull request by btazi**

### CODE REVIEW

File: app/assets/javascripts/activeadmin_addons/inputs/nested-select.js [hunk 1]
```diff
@@ -150,6 +150,9 @@ $(function() {
 
       if (!!parent) {
         var parentSelectorId = '#' + model + '_' + parent;
+				if (!$(parentSelectorId).length) {
+					parentSelectorId = container.find('*[id*='+ parent +']')[0];
+				}
```
Comment (monkeyci) [inline app/assets/javascripts/activeadmin_addons/inputs/nested-select.js]: Expected indentation of 0 spaces but found 4 tabs  indent

File: app/assets/javascripts/activeadmin_addons/inputs/nested-select.js [hunk 2]
```diff
@@ -150,6 +150,9 @@ $(function() {
 
       if (!!parent) {
         var parentSelectorId = '#' + model + '_' + parent;
+				if (!$(parentSelectorId).length) {
+					parentSelectorId = container.find('*[id*='+ parent +']')[0];
```
Comment (monkeyci) [inline app/assets/javascripts/activeadmin_addons/inputs/nested-select.js]: Expected indentation of 2 spaces but found 5 tabs  indent<br>Infix operators must be spaced                     space-infix-ops

File: app/assets/javascripts/activeadmin_addons/inputs/nested-select.js [hunk 3]
```diff
@@ -150,6 +150,9 @@ $(function() {
 
       if (!!parent) {
         var parentSelectorId = '#' + model + '_' + parent;
+				if (!$(parentSelectorId).length) {
```
Comment (monkeyci) [inline app/assets/javascripts/activeadmin_addons/inputs/nested-select.js]: Expected indentation of 8 spaces but found 4 tabs  indent

**Comment by ldlsegovia** [comment]
@btazi please follow the following [rules](https://github.com/platanus/activeadmin_addons/blob/master/docs/CONTRIBUTING.md#rules) if you want this merged. Thanks! :)

**Pull request closed without merging**